lib/src/db/model_db_impl.dart

```diff
@@ -439,7 +450,12 @@ class _ModelDescription<T extends Model> {
           '${entity.key.elements.last.kind} (property name: $propertyName)');
     }
 
-    mirror.setField(mirrors.MirrorSystem.getSymbol(fieldName), value);
+    try {
+      mirror.setField(mirrors.MirrorSystem.getSymbol(fieldName), value);
+    } catch (error) {
```

Comment:
nit. Could we be more specific about what we are catching?

Reply:
Done